yarn.lock

```diff
@@ -655,6 +655,758 @@
     lodash "^4.17.13"
     to-fast-properties "^2.0.0"
 
+"@evocateur/libnpmaccess@^3.1.2":
```

Comment:
@marcdumais-work could you check MIT compatibility please

Reply:
I meant to run https://github.com/eclipse-theia/theia/wiki/Registering-CQs#tools-needed on all dependencies.

Reply:
ISC is a license [approved by the Foundation](https://www.eclipse.org/legal/licenses.php#approved), so this specific dependency itself looks ok. 

However the process we are asked to follow is more thorough - it checks recursively the production dependencies, in case a dependency of a dependency is not ok, license-wise. As well there are some production dependency version changes in this PR, so worth a full check to be safe. 

@kittakos, would you give it a try, following the instructions linked above?  I can help if you encounter issues or are uncertain about something. The goal is to have all committers become familiar with this and run whenever new or updated prod dependencies are introduced. Once you get a hand of it, it takes ~10 minutes.

Reply:
> would you give it a try,

Sure.